tests/testPosixStorage.py

```diff
@@ -139,7 +139,14 @@ def testAbsolutePath(self):
         f.close()
         del f
         f = storage.getLocalFile('foo.txt')
-        self.assertIsInstance(f, file)
+        try:
+            fileType = file
+        except NameError:
+            from io import IOBase
+            fileType = IOBase
```

Comment:
Also, if you do need to still assign the class to a variable, please use a variable name that starts with a capital letter.